net/strongswan/files/swanctl.init

```diff
@@ -443,21 +512,62 @@ config_remote() {
 		;;
 	esac
 
+	local ike_proposal
+	parse_ike_proposal "$1" ike_proposal
```

Comment:
I don't want to move it here.  That doesn't reflect the fact that `swanctl.conf` can have one IKE definition but multiple child-SA's each with its own ESP parameters.